ssl/src/main/java/org/wildfly/security/ssl/X509RevocationTrustManager.java

```diff
@@ -245,6 +270,18 @@ public Builder setCrlStream(InputStream crlStream) {
             return this;
         }
 
+        /**
+         * Set the input streams pointing to certificate revocation lists (may be an empty list). The streams will be
+         * automatically closed after the invocation
+         *
+         * @param crlStreams the input streams
+         * @return this Builder for subsequent changes
+         */
+        public Builder setCrlStreams(List<InputStream> crlStreams) {
```

Comment:
I think the `setCrlStream(InputStream crlStream)` method above could be modified to initialize `this.crlStreams` instead of `this.crlStream`. The `crlStream` variable could then be removed and then the corresponding `if` block in the constructor could be simplified.

Reply:
fixed